lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -18,15 +18,35 @@ class AmazonMarketplaceWebService < Service
     XML_FAILURE_RESPONSE = { :success => FAILURE }.freeze
 
     ENDPOINTS = {
-      :ca => 'mws.amazonservices.ca',
+      :au => 'mws.amazonservices.com.au',
+      :br => 'mws.amazonservices.com',
+      :ca => 'mws.amazonservices.com',
       :cn => 'mws.amazonservices.com.cn',
-      :de => 'mws-eu.amazonservices.ca',
-      :es => 'mws-eu.amazonservices.ca',
-      :fr => 'mws-eu.amazonservices.ca',
-      :it => 'mws-eu.amazonservices.ca',
+      :de => 'mws-eu.amazonservices.com',
+      :es => 'mws-eu.amazonservices.com',
+      :fr => 'mws-eu.amazonservices.com',
+      :in => 'mws.amazonservices.in',
+      :it => 'mws-eu.amazonservices.com',
       :jp => 'mws.amazonservices.jp',
-      :uk => 'mws-eu.amazonservices.ca',
-      :us => 'mws.amazonservices.com'
+      :mx => 'mws.amazonservices.com',
+      :uk => 'mws-eu.amazonservices.com',
+      :us => 'mws.amazonservices.com',
+    }.freeze
+
+    MARKETPLACE_IDs = {
+      :au => 'A39IBJ37TRP1C6',
+      :br => 'A2Q3Y263D00KWC',
+      :ca => 'A2EUQ1WTGCTBG2',
+      :cn => 'AAHKV2X7AFYLW',
+      :de => 'A1PA6795UKMFR9',
+      :es => 'A1RKKUPIHCS9HS',
+      :fr => 'A13V1IB3VIYZZH',
+      :in => 'A21TJRUUN4KGV',
+      :it => 'APJ6JRA9NG5V4',
+      :jp => 'A1VC38T7YXB528',
+      :mx => 'A1AM78C64UM0Y8',
+      :uk => 'A1F83G8C2ARO7P',
```

Comment:
The endpoint docs specify `GB` as the country code for the UK marketplace, but it seems we've chosen `UK` here instead. Just calling out the discrepancy as it's the only place where we differ from the Amazon country code to marketplace id mapping.

Reply:
since we were using `UK` before, I will add another line for GB. That way both are supported